Add New Themed Beatmap Packs news post (#2667)
ppy/osu-wiki: news/2019-11-14-beatmap-pack-management-update-1.md

---
layout: post
title: Introducing Beatmap Pack Management
date: 2019-11-14 14:00:00 +0000
---

An update to the Beatmap Pack Management project, including new pack releases!

![](/wiki/shared/news/banners/beatmap-packs.jpg)

> Our project is called Beatmap Pack Management and we are focused on development of the Beatmap Packs. Many of you might not even know who we are or what we are doing, so I'd like to present some information about us:

Reviewer: ```suggestion
Our project, called Beatmap Pack Management, is here to make more Beatmap Packs focused on a variety of themes for the community to enjoy! For those unaware, here's some more information about us:
```

> I've created a small group of experienced users who are interested in working on handmade charts dedicated to specific themes, ideas, or beatmaps which share similarities. Since July 2019, we've been actively working to provide new packs of interesting themes. Interest in this project is very high and some noteworthy members of the community have joined the group to help us by contributing their ideas to the project. We already have a few things to present to you, and I am sure you'll like them.

Reviewer: ```suggestion
I've created a small group of experienced users interested in working on packs of beatmaps dedicated to specific themes, similarities, and ideas. We have been actively working since July 2019 to provide new and interesting packs to play! Since we began the Beatmap Pack Management initiative, interest in this project has been very high and some noteworthy members of the community have also joined the group to help us by contributing their ideas to the project. We already have a few things to present to you which I'm sure you'll like. Be sure to check out the packs below!
```

### Alternator Style packs

> *created by [lain](https://osu.ppy.sh/users/1650010), [Uberzolik](https://osu.ppy.sh/users/1314547)*  

Reviewer: ```suggestion
*created by [Aireu](https://osu.ppy.sh/users/1650010), [Uberzolik](https://osu.ppy.sh/users/1314547)*  
```

*written by [Stefan](https://osu.ppy.sh/users/626907)*  
Download [Ranked](https://osu.ppy.sh/beatmaps/packs/1699) | [Loved](https://osu.ppy.sh/beatmaps/packs/1700)  

> lain and Uberzolik were passionate enough to work on the very first mapping style specific beatmap packs ever released. These two packs are focused on beatmaps where alternating is essential. lain has worked on the ranked version while Uberzolik took care of the Loved version. If you need maps to practice alternating or you are good at it already and want to challenge yourself, you should check these packs out!

Reviewer: ```suggestion
Aireu and Uberzolik were passionate enough to work on the very first mapping style specific beatmap packs ever released. These two packs are focused on beatmaps where alternating is essential. Aireu worked on the ranked version while Uberzolik took care of the Loved version. If you need maps to practice alternating, or you are good at it already and want to challenge yourself, you should check these packs out!
```

### The Embodiment of Scarlet Devil pack

*created by [celerih](https://osu.ppy.sh/users/4696296), [Uberzolik](https://osu.ppy.sh/users/1314547), [Venix](https://osu.ppy.sh/users/5999631), and [Xilver15](https://osu.ppy.sh/users/3099689)*  
*written by [celerih](https://osu.ppy.sh/users/4696296)*  
[Download](https://osu.ppy.sh/beatmaps/packs/1729)  

> Available to download now, the pack is the first of the Touhou Game Packs, where a selection of various Touhou maps from a specific game is picked and curated just for you! This pack contains beatmaps from the first non PC-98 Touhou game and the sixth game in the series - Embodiment of Scarlet Devil. The pack aims to offer a variety of styles and themes, ranging from 2008 to 2018 to offer something enjoyable for everyone. Packs for future games are currently in progress and should be released within the next few weeks. Enjoy!

Reviewer: ```suggestion
Available to download now, the pack is the first of the Touhou Game Packs, where a selection of various Touhou maps from a specific game is picked and curated just for you! This pack contains beatmaps from the first non PC-98 Touhou game and the sixth game in the series - Embodiment of Scarlet Devil. The pack aims to offer a variety of styles and themes, ranging from 2008 to 2018 to offer something enjoyable for everyone. Packs for future games are currently in progress and should be released in the near future. Enjoy!
```

Each of the dans will contain 20 levels of increasing difficulty. In order to complete a level, the player must reach the requirements on the maps presented, like reaching a certain accuracy threshold or attaining a full combo. For now, the release of dan levels is purely for fun, a way to practice, or test yourself, but a more official implementation is planned for the future. 😉

The first of the dans is available to download now on the Beatmap Packs listing. The skill tested in this pack is aim and will surely push your circle clicking to the limits. The requirements for this dan are quite simple: you need to achieve full combo on 10 out of 11 maps in the dan to progress to the next one. Once again, this for now is just for your own practice, or to compare yourself with your friends - there is no implementation for keeping track of levels as for now.

If you wish to help with playtesting future packs and want to stay up to date, you can join the official [osu!dan Discord server](https://discord.gg/b7cVmGd) now!

### October Loved packs

*written by [Stefan](https://osu.ppy.sh/users/626907)*

> As it is time again to bundle a few beatmaps, there are some new packs for loved beatmaps available since mid October. I want to take the chance to explain why I did not bundle the latest maps from Loved already: Since there is always a small chance that a beatmap can become unloved it's a bit tedious to keep track of these things. I decided to give maps at least a minimum of 10 months to stay in Loved, and to bundle them afterwards into packs. While it's no guarantee they will stay in Loved forever, it works for me and I'll go by that for now. Changes can always happen though and I am open for alternatives.

Reviewer: ```suggestion
The time came once again to bundle a few beatmaps, with some new Loved beatmap packs being made available in mid October. That said, I want to take this chance to explain why Loved beatmaps do not get bundled into packs right away. Since there is always a small chance that a beatmap can become unloved, I decided to give maps at least a minimum of 10 months to stay in Loved, and then afterwards bundle the Loved beatmaps into packs. While it's no guarantee they will stay in Loved forever, it works for me and I'll go by that for now. Changes can always happen though and I am open for alternatives!
```

Reviewer: mostly putting it in past tense since this news post got delayed so much that saying "it is time" isn't accurate anymore (unless you decide to additionally publish november and december loved packs now)

### What else to expect

> While I believe we're doing very well already, we're just about to start. Currently, I'm the only active person that coordinates the uploads, as well as the accessibility and maintenance of the packs so it's a little slow. But I'm glad I could help a lot of people so far and that I have received so much great feedback. If you are interested in taking a look at what we are working on, or if you have to report an error/bug in a pack, you can join our project server and spectate our doings. Alternatively, if you are interested to work in the group you can join it too! Please check out the [forum thread](https://osu.ppy.sh/community/forums/topics/933379) for information on how to get to the server.

Reviewer: ```suggestion
While I believe we're doing very well already, we're just getting started. Currently, I'm the only active person that coordinates the uploads, as well as the accessibility and maintenance of the packs so it's a little slow. But I'm glad I could help a lot of people so far and that I have received so much great feedback. If you are interested in taking a look at what we are working on, or if you have to report an error/bug in a pack, you can join our project server and spectate our doings. Alternatively, you can even join the group if you're interested! Please check out the [forum thread](https://osu.ppy.sh/community/forums/topics/933379) for information on how to get to the server.
```
